backend/chainlit/slack_websocket_test.py

```diff
@@ -0,0 +1,18 @@
+# This is a simple example to test the slack websocket handler.
+# To initiate the websocket dont forget to set the variables:
+#   - SLACK_BOT_TOKEN
+#   - SLACK_SIGNING_SECRET
+#   - SLACK_WEBSOCKET_TOKEN <- this one dictates if websocket or http handler
+
+from chainlit import Message, on_message, user_session
+
+
+@on_message
+async def main(message: Message):
+    client_type = user_session.get("client_type")
+    if client_type == "slack":
+        user_email = user_session.get("user").metadata.get("email")
+        print(f"Received a message from: {user_email}")
+        await Message(
+            content=f"Hi {user_email}, I have received the following message:\n{message.content}",
+        ).send()
```

Comment:
It's really great that you've added this script to facilitate testing, but this really isn't the place for it!

Rather, we have several designated places for such functionality:
1. The chainlit [cookbook](https://github.com/Chainlit/cookbook/tree/main). It would be great to have a minimal example of slack bot integration there! In addition, it's also a great guidance for users to get started with this!
2. Unit tests. It would be absolutely amazing to have a unit test for Slack -- both the legacy behaviour as well as websocket behaviour. 
3. E2E tests, they live in `https://github.com/Chainlit/chainlit/tree/main/cypress/e2e`. But I don't think it's relevant for Slack as I don't think there's a browser UX in this case!

The way to test this is to [monkeypatch](https://docs.pytest.org/en/stable/how-to/monkeypatch.html) `chainlit.slack.app.AsyncSocketModeHandler` and `AsyncApp` with `spec=AsyncSocketModeHandler` and `AsyncApp`, respectively. 

You could then test:
1. Whether all Slack functions are properly called by calling assertions on the mock.
2. Where relevant, manually call Slack's callback hooks _as if_ the user sends messages. 

For now, just having _any_ testing for Slack integration would be a huuuuuuge contrib. If you can't make it, that's okay, just remove this test file and file a PR for the cookbook with it, as well as a PR for the docs discussing this new behaviour.